editor/editor_node.cpp

```diff
@@ -6664,13 +6664,16 @@ int EditorNode::execute_and_show_output(const String &p_title, const String &p_p
 
 	eta.execute_output_thread.start(_execute_thread, &eta);
 
+	print_line("\nGradle Build Started:\n");
```

Comment:
Use sentence case capitalization:

```suggestion
	print_line("\nGradle build started:\n");
```